### PR TITLE
Adding automerge

### PR DESCRIPTION
**Pull request by dennisc87**
There are no tests that can run without a Github token, so the only required check is Gitguardian.

I have edited the branch protection rules by hand to allow automerge.

Let me know if there was a reason it was not added here before now :)

### CODE REVIEW

**Review by Andrei-Predoiu — APPROVED** [review]
lgtm. no reason, i just added it to repos as they got dependabot prs. 
Remember to check that the automerge label is enabled on the repo

**Comment by Gaardsholt** [comment]
Maybe we should just add a CircleCI workflow that runs the unit tests?
just for good measure

**Comment by dennisc87** [comment]
> Maybe we should just add a CircleCI workflow that runs the unit tests? just for good measure

**Comment by dennisc87** [comment]
> Maybe we should just add a CircleCI workflow that runs the unit tests? just for good measure

Made it with Github actions, and the built in short lived GITHUB_TOKEN. What do you feel about this, instead of setting up CCI and Secrets retrieval?

File: .github/workflows/go_test.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      with:
+        go-version: '1.21'
+
+    - name: Build
+      run: go build
+
+    - name: Test
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: go test
```
Comment (Gaardsholt) [inline .github/workflows/go_test.yml:29]:
```suggestion
      run: go test ./...
```

File: .github/policy.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+              - es has approved
+  disapproval:
+    requires:
+      organizations:
+        - "bestseller"
+approval_rules:
+  - name: tests passed
+    if:
+      has_successful_status:
+        - "GitGuardian Security Checks"
```
Comment (Gaardsholt) [inline .github/policy.yml:20]: Shouldn't we add the "Go Test" here, so it won't merge until it has completed with success?

Comment (dennisc87) [reply .github/policy.yml:20]: Yes

**Pull request merged**